spec/features/applicant_page_spec.rb

```diff
@@ -0,0 +1,173 @@
+require "rails_helper"
+
+RSpec.describe "Applicant Page" do
+  let(:income_header) { "What income does your client receive?" }
+  let(:property_header) { "Does your client own the home they live in?" }
+  let(:applicant_header) { "About your client" }
+  let(:estimate_id) { SecureRandom.uuid }
+  let(:mock_connection) { instance_double(CfeConnection, create_assessment_id: estimate_id) }
+
+  describe "errors" do
+    before do
+      allow(CfeConnection).to receive(:connection).and_return(mock_connection)
+      visit "/estimates/new"
+
+      %i[over_60 dependants partner passporting employed].reject { |f| f == field }.each do |f|
+        select_applicant_boolean(f, true)
+      end
+      click_on "Save and continue"
+    end
+
+    context "with over_60" do
```

Comment:
I'm a little slow in grokking specs, and I didn't understand at first why the expectation was that there would be an error in these specs. To make it easier for me to tell what's going on, could we make the context string something like `"when the 'over 60' question is omitted"`?